include/umf/memory_provider.h

```diff
@@ -188,12 +188,13 @@ umfMemoryProviderOpenIPCHandle(umf_memory_provider_handle_t hProvider,
 /// @brief Close an IPC memory handle.
 /// @param hProvider [in] handle to the memory provider.
 /// @param ptr [in] pointer returned by umfMemoryProviderOpenIPCHandle function.
+/// @param size [in] size of the memory.
```

Comment:
Maybe "`size of the memory address range`" like above in `umfMemoryProviderGetIPCHandle`?

Reply:
Done